socorro/tests/external/es/base.py

```diff
@@ -1,186 +0,0 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at https://mozilla.org/MPL/2.0/.
-
-from datetime import timedelta
-from distutils.version import LooseVersion
-from functools import wraps
-import random
-import uuid
-
-from configman import ConfigurationManager, environment
-import pytest
-
-from socorro.external.es.connection_context import ConnectionContext
-from socorro.external.es.crashstorage import ESCrashStorage
-from socorro.lib.libdatetime import utc_now
-
-
-DEFAULT_VALUES = {
-    "resource.elasticsearch.elasticsearch_index": "testsocorro_%W",
-    "resource.elasticsearch.elasticsearch_index_regex": "^testsocorro.*$",
-    "resource.elasticsearch.elasticsearch_timeout": 10,
-}
-
-
-CRON_JOB_EXTA_VALUES = {"resource.elasticsearch.backoff_delays": [1]}
-
-
-def minimum_es_version(minimum_version):
-    """Skip the test if the Elasticsearch version is less than specified
-
-    :arg minimum_version: string; the minimum Elasticsearch version required
-
-    """
-
-    def decorated(test):
-        """Decorator to only run the test if ES version is greater or equal than specified"""
-
-        @wraps(test)
-        def test_with_version(self):
-            """Only run the test if ES version is not less than specified"""
-            actual_version = self.conn.info()["version"]["number"]
-            if LooseVersion(actual_version) >= LooseVersion(minimum_version):
-                test(self)
-            else:
-                pytest.skip()
-
-        return test_with_version
-
-    return decorated
-
-
-class TestCaseWithConfig:
-    """A simple TestCase class that can create configuration objects"""
-
-    def setup_method(self):
-        pass
-
-    def teardown_method(self):
-        pass
-
-    def get_tuned_config(self, sources, extra_values=None):
-        if not isinstance(sources, (list, tuple)):
-            sources = [sources]
-
-        config_definitions = []
-        for source in sources:
-            conf = source.get_required_config()
-            config_definitions.append(conf)
-
-        values_source = {}
-        if extra_values:
-            values_source.update(extra_values)
-
-        config_manager = ConfigurationManager(
-            config_definitions,
-            app_name="testapp",
-            app_version="1.0",
-            app_description="Elasticsearch integration tests",
-            values_source_list=[environment, values_source],
-            argv_source=[],
-        )
-
-        return config_manager.get_config()
-
-
-class ElasticsearchTestCase(TestCaseWithConfig):
```

Comment:
I moved most of this to an `es_helper` fixture that's more convenient to use. That's in `socorro/tests/conftest.py`.